examples/bootstrap4/templates/table.html

```diff
@@ -43,4 +43,7 @@ <h2>Table with custom actions</h2>
                         ('like_message', [('message_id', ':id')])
                     )
                 ]) }}
+<h2>Table without SQLAlchemy</h2>
+<pre>{% raw %}{{ render_table(data, titles) }}{% endraw %}</pre>
```

Comment:
It's better to add a description of what the data looks like. For example:

```suggestion
<p>The <pre>data</pre> contains a list of dict: <pre>[{'field1': 'value', 'field2': 'value', ...}, {...}]</pre></p>
<pre>{% raw %}{{ render_table(data, titles) }}{% endraw %}</pre>
```

Reply:
This change also needs to be mirrored in the template for Bootstrap 5.